.envrc

```diff
@@ -10,6 +10,8 @@ export ENVIRONMENT=local
 export CLIENT_HOSTNAME=http://localhost
 export CLIENT_PORT=3000
 export CLIENT_ADDRESS=$CLIENT_HOSTNAME:$CLIENT_PORT
+export API_PORT=8080
+export REACT_APP_API_ADDRESS=$CLIENT_HOSTNAME:$API_PORT/api/v1
```

Comment:
Minor question: Will there ever come a time when we need to access /api/v1 for some endpoints and /api/v2 for others? If so, then maybe remove "/v1" or "/api/v1" from the environment variable? But that's also something that could be modified later if need be.

Reply:
v1 -> v2 would be a major feature that can break backwards compatibility. Since we don't have an app that requires demanding uptime and only consume our own API, I don't anticipate a huge need to run both at the same time, but it's possible. We can always edit the env variables then. It's possible we'll be running separate builds too.